packages/model-viewer/src/three-components/Renderer.ts

```diff
@@ -55,15 +58,31 @@ const DEFAULT_LAST_STEP = 3;
  * the texture.
  */
 export class Renderer extends EventDispatcher {
-  private static _singleton = new Renderer({debug: isDebugMode()});
+  private static _powerPreference = DEFAULT_POWER_PREFERENCE;
+  private static _singleton : Renderer;
+
+  static set powerPreference(powerPreferenceOption:string) {
+    this._powerPreference = powerPreferenceOption || DEFAULT_POWER_PREFERENCE;
+  }
+  
+  static get powerPreference() {
+    return this._powerPreference;
+  }
```

Comment:
If we have both a getter and a setter that don't have logic in them, we might as well just have a public member and skip the boiler plate.

Reply:
Also, is it actually possible to set the powerPreference outside of the constructor and have it affect anything? If not, then maybe it shouldn't be a member at all.

Reply:
So is it only recommended to have setters/getters only if they have logic in them?

Reply:
I'm not quiet sure what you mean by your other comment (having powerPreference outside of the constructor)

Reply:
I would say use setters and getters when they are necessary. Likewise, rather than having this state in two separate static variables, perhaps just use the ModelViewer one directly so you don't need it here too.

I see now what you did with the singleton, though it does seem to have changed some behavior since a test is failing now. Perhaps it needs to be prodded into being instantiated?

Reply:
I updated the setters/getters, but I'm not convinced why my change should break the test